pkg/generator/code_generator_test.go

```diff
@@ -67,6 +73,13 @@ func TestGenerators(t *testing.T) {
 		assert.Equal(t, expectedFormRequestGoCode, result, result)
 	})
 
+	t.Run("java form HTTP request", func(t *testing.T) {
+		result, err := generator.GetCodeGenerator("java").Generate(nil, formRequest)
+		print(result)
```

Comment:
You don't need this line because line 80 will print it if there are some errors happend.